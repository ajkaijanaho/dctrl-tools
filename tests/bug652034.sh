#!/bin/sh

set -e

${GREP_DCTRL:-grep-dctrl} -FFoo a bug652034.in
echo '###'
${GREP_DCTRL:-grep-dctrl} -FFoo b bug652034.in
echo '###'
${GREP_DCTRL:-grep-dctrl} -sFoo -FFoo b bug652034.in
