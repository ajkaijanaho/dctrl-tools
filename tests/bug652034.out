Foo: a
Foo: b

###
Foo: a
Foo: b

###
Foo: a
Foo: b
